spinalcordtoolbox/scripts/sct_get_centerline.py

```diff
@@ -79,6 +79,14 @@ def get_parser():
         default=0,
         help="Binary or soft centerline. 0 = binarized, 1 = soft. Only relevant with -method fitseg."
     )
+    optional.add_argument(
+        "-extrapolation",
+        metavar=Metavar.int,
+        type=int,
+        choices=[0, 1],
+        default=0,
+        help="Extrapolate beyond the segmentation boundaries. 0 = no extrapolation, 1 = extrapolation. Only relevant with -method fitseg."
```

Comment:
I would perhaps consider adding a note about the potentially erratic behavior when combined with higher-order interpolation methods (bspline, polyfit), as described in https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3531#issuecomment-1483347201

In other words, we could mention that `-extrapolate` works best with lower-order interpolation methods?